### PR TITLE
Add the "Admissions from supervision to prison by supervision level" chart

### LINKED ISSUE #1506: Create the "Admissions from supervision to prison by supervision level" chart
**Is your feature request related to a problem? Please describe.**
Create the "Admissions from supervision to prison by supervision level" chart for US_TN on the "Supervision to Prison" page.

metric should be called supervisionToPrisonPopulationBySupervisionLevel, and will be an instance of PopulationSnapshotMetric.
id: "supervisionToPrisonPopulationBySupervisionLevel",
tenantId: this.rootStore.currentTenantId,
compartment: "SUPERVISION",
sourceFilename: "supervision_to_prison_population_snapshot_by_dimension",
rootStore: this.rootStore,
dataTransformer: createSupervisionPopulationSnapshot,
accessor: supervisionLevel,
filters: {
    enabledFilters: [
      FILTER_TYPES.DISTRICT,
      FILTER_TYPES.MOST_SEVERE_VIOLATION,
      FILTER_TYPES.NUMBER_OF_VIOLATIONS,
    ],
    enabledMoreFilters: [
      FILTER_TYPES.SUPERVISION_TYPE,
      FILTER_TYPES.GENDER,
      FILTER_TYPES.AGE_GROUP,
      FILTER_TYPES.SUPERVISION_LEVEL
    ],
  },

Also enable the SUPERVISION_LEVEL filter for US_TN on all of the other "Supervision to Prison" charts.

The filter options for the SUPEVISION_LEVEL filter are:
  { label: "All", value: "all" },
  { label: "Minimum", value: "MINIMUM" },
  { label: "Medium", value: "MEDIUM" },
  { label: "Maximum", value: "MAXIMUM" }

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
Add any other context about the feature request here.

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

Added the "Admissions from supervision to prison by supervision level" chart and filter.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #1506 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jovergaag** [comment]
Hi @nasaownsky I don't have a fixture file with this dimension ready yet so I will hold off on review until that is set. With the new dimensions (filters) coming up in January the fixture files are getting to be too large for the FE to handle so I am working on a solution for that. Thank you for your patience while I get this sorted out!

You should already have the fixture files that you need to complete #1497 and #1500.

File: src/core/models/SupervisionPopulationSnapshotMetric.ts [hunk 1]
```diff
@@ -68,6 +68,9 @@ export default class SupervisionPopulationSnapshotMetric extends PathwaysMetric<
           (this.id === "supervisionToPrisonPopulationByNumberOfViolations"
             ? !["ALL"].includes(record.numberOfViolations)
             : numberOfViolations.includes(record.numberOfViolations)) &&
+          (this.id === "supervisionToPrisonPopulationBySupervisionLevel"
+            ? !["ALL"].includes(record.supervisionLevel)
+            : !["ALL"].includes(record.supervisionLevel)) && // change to supervisionLevel.includes(record.supervisionLevel)) when the data is ready
```
Comment (jovergaag) [inline src/core/models/SupervisionPopulationSnapshotMetric.ts:73]: The data in the `jovergaag/demo-filters` branch is ready for this now! If you base this off of that branch you should see a working chart.

File: src/core/models/utils.ts [hunk 2]
```diff
@@ -106,6 +106,7 @@ export function createSupervisionPopulationSnapshot(
         numberOfViolations: record.number_of_violations,
         lengthOfStay: record.length_of_stay,
         totalPopulation: parseInt(record.total_population),
+        supervisionLevel: record.supervision_level,
```
Comment (jovergaag) [inline src/core/models/utils.ts:109]: You will also need to add this to `createSupervisionPopulationTimeSeries`

File: src/core/utils/filterOptions.ts [hunk 3]
```diff
@@ -301,6 +301,24 @@ export const PopulationFilterOptions: PopulationFilters = {
     },
     enabledViews: [CORE_VIEWS.facilities],
   },
+  [FILTER_TYPES.SUPERVISION_LEVEL]: {
+    type: FILTER_TYPES.SUPERVISION_LEVEL,
+    title: "Supervision Level",
+    setFilters: setFilters(FILTER_TYPES.SUPERVISION_LEVEL),
+    options: [
+      { label: "All", value: "all" },
```
Comment (jovergaag) [inline src/core/utils/filterOptions.ts:309]: The value here should be all caps `ALL`

**Review by jovergaag — CHANGES_REQUESTED** [review]

**Comment by nasaownsky** [comment]
@jovergaag Do you plan to merge `jovergaag/demo-filters` into main? I have some issues with rebasing this branch, and my old files from `demo-data` is gone now.

**Comment by jovergaag** [comment]
> @jovergaag Do you plan to merge `jovergaag/demo-filters` into main? I have some issues with rebasing this branch, and my old files from `demo-data` are gone now.

I will merge it Tuesday afternoon after the data platform production deploy. There is a schema change in that branch that relies on the deploy, so to keep `main` deployable I will wait for that data platform deploy.

I will do the rebase for you.

**Review by jovergaag — APPROVED** [review]

**Pull request merged**